_inc/client/components/module-settings/index.jsx

```diff
@@ -460,7 +460,7 @@ export let VerificationToolsSettings = React.createClass( {
 
 					<div className="dops-card">
 						<FormLabel>
-							<FormLegend>Google</FormLegend>
+							<FormLegend>{ __( 'Google' ) }</FormLegend>
```

Comment:
All these do not have to be translatable because they're brand names that aren't meant to be translated and they will unnecessary add strings to translate so please leave them as they were before.